core/src/main/scala/tofu/package.scala

```diff
@@ -2,13 +2,25 @@ import cats.effect.Bracket
 import cats.{ApplicativeError, MonadError}
 
 package object tofu {
-  type HasContext[F[_], C]          = Context[F] { type Ctx    = C }
-  type HasLocal[F[_], C]            = Local[F] { type Ctx      = C }
+  type HasContext[F[_], C] = Context[F] { type Ctx = C }
+  object HasContext {
```

Comment:
I fill those belong to Errors.scala

Reply:
@odomontois Why do you think summoning instances belongs to Errors file?

Reply:
package objects are nasty things in terms of refactoring, bytecode, binary compatibility
I'd like to have them as small as possible, so all the things, that are defineable in the package itself, should be defined in the package itself, I feel